packages/core/src/components/RadioButton/RadioButtonGroup.tsx

```diff
@@ -58,16 +57,10 @@ export const SEED_DATA = {
   layout: {},
   props: {
     direction: createDirectionProp(),
-    value: createTextProp({
-      label: "value",
-      description: "Currently selected value of the radio button group",
-      defaultValue: null,
-      required: true,
-    }),
     fieldName: createFieldNameProp({
       handlerPropName: "onValueChange",
-      valuePropName: "radioButtonGroupValue",
```

Comment:
This was creating a prop called `radioButtonGroupValue` which is not consumed by this component